pkgs/development/interpreters/s9fes/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchurl, ncurses, buildPackages }:
+
+let
+  isCrossCompiling = stdenv.hostPlatform != stdenv.buildPlatform;
+in
+
+stdenv.mkDerivation rec {
+  pname = "s9fes";
+  version = "20181205";
+
+  src = fetchurl {
+    url = "http://www.t3x.org/s9fes/s9fes-${version}.tgz";
```

Comment:
```suggestion
     url = "https://www.t3x.org/s9fes/s9fes-${version}.tgz";
``

Reply:
Not sure if another commit is necessary but it drives me nuts now

Reply:
Oops I thought it was a whitespace issue. Nevermind, this is fine.

Reply:
The whitespace is probably there because github declined me the suggestion feature.